sdk/digitaltwins/azure-digitaltwins-core/azure/digitaltwins/core/_generated/aio/operations/_digital_twin_models_operations.py

```diff
@@ -450,7 +450,7 @@ async def delete(
         if digital_twin_models_delete_options is not None:
             _traceparent = digital_twin_models_delete_options.traceparent
             _tracestate = digital_twin_models_delete_options.tracestate
-        api_version = "2020-10-31"
+        api_version = self._config.api_version
```

Comment:
This (and others) seem to be in generated files... Is it intentional that we would want to modify these? This file has a comment at the beginning indicating that it shouldn't be modified directly.